Internal server error when selecting certain extensions

**Describe the bug**
I have discovered this error in two cases:
Case 1: Select Kotlin and Quarkus Extension for Spring Web API, then click "Generate your application"
Case 2: Select Scala and Quarkus Extension for Spring Web API, then click "Generate your application"

![image](https://user-images.githubusercontent.com/20326645/69758762-63dfca80-112e-11ea-8c6b-1b35fec6f275.png)
 
**Expected behavior**
Project should download. Or maybe its not meant for these two extensions to be used in the same project? Although, I am able to add these two extensions in the same project using the command line just fine.

**Actual behavior**
Internal server error

![image](https://user-images.githubusercontent.com/20326645/69758818-92f63c00-112e-11ea-94e6-93750d2c4185.png)

hello, I was able to reproduce this problem locally.
```
Caused by: java.io.IOException: Failed to locate templates/basic-rest/scala/spring-controller-template.ftl on the classpath
	at io.quarkus.platform.descriptor.loader.json.ClassPathResourceLoader.loadResource(ClassPathResourceLoader.java:24)
	at io.quarkus.platform.descriptor.loader.json.impl.QuarkusJsonPlatformDescriptor.getTemplate(QuarkusJsonPlatformDescriptor.java:108)
	... 92 more
```
what should be the expected behavior? to return a 422 error saying that selected framework X is not supported by langague Y? Or to create the template?

--- a/devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/kotlin/spring-controller-template.ftl
+++ b/devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/kotlin/spring-controller-template.ftl
@@ -1,0 +1,15 @@
+package ${package_name};
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PathVariable;
+
+
+@RestController
+@RequestMapping("${path}")
+class ${class_name} {
+
+    @GetMapping
+    fun hello() = "hello"
+}

--- a/devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/spring-controller-template.ftl
+++ b/devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/spring-controller-template.ftl
@@ -1,0 +1,12 @@
+package ${package_name};
+
+import org.springframework.web.bind.annotation.{GetMapping, RequestMapping, RestController, PathVariable};
+
+
+@RestController
+@RequestMapping(Array[String]("${path}"))
+class ${class_name} {
+
+    @GetMapping
+    def hello() = "hello"
+}

